### PR TITLE
Version 2.0.0-rc1 release

### PR DESCRIPTION
**Pull request by garg-mudit**
### Breaking Changes
- Updated C# SDK dependencies.
    - Microsoft.CSharp: Version bumped from 4.5.0 to 4.7.0.
    - Newtonsoft.Json: Version bumped from 13.0.1 to 13.0.3.
    - System.ComponentModel.Annotations: Version bumped from 4.5.0 to 5.0.0.
    - Microsoft.IdentityModel.Protocols: Version bumped from 5.4.0 to 7.3.1.
    - System.IdentityModel.Tokens.Jwt: Version bumped from 5.4.0 to 7.3.1.
    - BouncyCastle.Cryptography: Version bumped from 2.2.1 to 2.3.0.
### Changed
- Updated the SDK release version.

### CODE REVIEW

File: sdk/src/DocuSign.Admin/Api/AccountsApi.cs [hunk 1]
```diff
@@ -342,10 +342,16 @@ public ApiResponse<MemberGroupsResponse> GetGroupsWithHttpInfo(Guid? organizatio
                 if (options.end != null) localVarQueryParams.Add("end", this.ApiClient.ParameterToString(options.end)); // query parameter
             }
 
+            // authentication (docusignAccessCode) required
```
Comment (shan285) [inline sdk/src/DocuSign.Admin/Api/AccountsApi.cs:345]: @garg-mudit This is getting added in multiple places now.

**Review by shan285 — CHANGES_REQUESTED** [review]
Auth Method is added to all APIs now. Can we look into it.

File: sdk/src/DocuSign.Admin/Api/AccountsApi.cs (hunk 1 above)
Comment (garg-mudit) [reply sdk/src/DocuSign.Admin/Api/AccountsApi.cs:345]: Ticket [DCM-11094](https://docusign.atlassian.net/browse/DCM-11094) is present to check the same. 
As the code is not a breaking change, thus its okay keep it for now.

**Review by shan285 — APPROVED** [review]
LGTM

**Pull request merged**